src/eva/vision/data/transforms/normalization/clamp.py

```diff
@@ -0,0 +1,43 @@
+"""Image clamp transform."""
+
+import functools
+from typing import Any, Dict, Tuple
+
+import torch
+import torchvision.transforms.v2 as torch_transforms
+from torchvision import tv_tensors
+from typing_extensions import override
+
+
+class Clamp(torch_transforms.Transform):
+    """Clamps all elements in input into a specific range."""
+
+    def __init__(self, out_range: Tuple[int, int]) -> None:
+        """Initializes the transform.
+
+        Args:
+            out_range: The lower and upper bound of the range to
+                be clamped to.
+        """
+        super().__init__()
+
+        self._out_range = out_range
+
+    @functools.singledispatchmethod
+    @override
+    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
```

Comment:
Nit: Would it make sense to use more specific type hints for the return type?

Reply:
maybe also `inpt` ->  `input`

Reply:
got it from the `torchvision` source code - to be in line with them. `data` is maybe a good alternative but it depends how close we want to follow torchvision

@roman807 we can not use `input` because is a python key word